armi/physics/neutronics/tests/test_crossSectionManager.py

```diff
@@ -392,6 +395,22 @@ def test_addXsGroupsFromBlocks(self):
         self.assertEqual(len(blockCollectionsByXsGroup), 4)
         self.assertIn("AB", blockCollectionsByXsGroup)
 
+    def test_calcWeightedBurnup(self):
+        self.blockList[1].p.percentBu = 3.1
+        self.blockList[2].p.percentBu = 10.0
```

Comment:
maybe assert they all have the same volume or weight factor  since we need to know that to know its a simple avg

Reply:
I added an `assertEqual` to confirm `self.blockList[1]` and `self.blockList[2]` have the same weighting factor.